cccatalog-api/cccatalog/api/views/image_views.py

```diff
@@ -61,6 +61,8 @@ class SearchImages(APIView):
                          responses={
                              200: ImageSearchResultsSerializer(many=True),
                              400: InputErrorSerializer,
+                             500: 'relation "content_provider" does not \
```

Comment:
An error `500` is a generic server error, it won't necessarily be this specific one.